index.d.ts

```diff
@@ -73,7 +74,115 @@ declare namespace Eris {
   type MessageContent = string | AdvancedMessageContent;
   type MFALevel = 0 | 1;
   type PossiblyUncachedMessage = Message | { channel: TextableChannel | { id: string; guild?: Uncached }; guildID?: string; id: string };
-  type InteractionType = 1 | 2;
+
+  // Interaction
+  type InteractionDataOptions = InteractionDataOptionsSubCommand | InteractionDataOptionsSubCommandGroup | InteractionDataOptionsWithValue;
+  type InteractionDataOptionsWithValue = InteractionDataOptionsString | InteractionDataOptionsInteger | InteractionDataOptionsBoolean | InteractionDataOptionsUser | InteractionDataOptionsChannel | InteractionDataOptionsRole | InteractionDataOptionsMentionable | InteractionDataOptionsNumber;
+  interface InteractionDataOptionsSubCommand {
+    name: string;
+    type: Constants["ApplicationCommandOptionTypes"]["SUB_COMMAND"];
+    options: InteractionDataOptions[];
```

Comment:
```suggestion
    options?: InteractionDataOptions[];
```
The [docs seem to be a bit confusing about this](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-interaction-data-option-structure), but `options` appears to be optional in my testing. If the subcommand doesn't have any options, `options` here won't exist. This is also the case if the subcommand does accept non-required options, but none of them are provided in the interaction.

Basically: `options` can't be empty. It's either undefined, or it has something in it.